test/injected-script-test.js

```diff
@@ -1351,4 +1351,19 @@ describe("canjs-devtools-injected-script", () => {
 			assert.equal(resp.detail, "no component", "detail is the error message");
 		});
 	});
+
+	describe('addBreakpoints', () => {
+		it ('adds breakpoints', () => {
+			const breakpoints = devtools.addBreakpoints([{
```

Comment:
This test should probably show adding multiple breakpoints.

Reply:
Done!